redvanworkshop/website
# Martin case study MK-122

#### What's this PR do?

This adds the Martin Guitar Case Study

#### Where should the reviewer start?

Clients Page

#### How should this be manually tested?

Go to clients page, navigate to the Martin card and then check the martin case study page linked.

#### Any background context you want to provide?

I had to add an page link to the includes folder for the sfcc order management. If this needs any changes please check and let me know.

#### What are the relevant jira issue?

MK-122

#### Screenshots

![image](https://user-images.githubusercontent.com/79174401/119174810-0d0f3480-ba37-11eb-8170-427758b32cd3.png)
